tests/Functional/Anonymizer/Core/DateAnonymizerTest.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MakinaCorpus\DbToolsBundle\Tests\Functional\Anonymizer\Core;
+
+use MakinaCorpus\DbToolsBundle\Anonymization\Config\AnonymizationConfig;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizator;
+use MakinaCorpus\DbToolsBundle\Anonymization\Config\AnonymizerConfig;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\AnonymizerRegistry;
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Options;
+use MakinaCorpus\DbToolsBundle\Test\FunctionalTestCase;
+
+class DateAnonymizerTest extends FunctionalTestCase
+{
+    /** @before */
+    protected function createTestData(): void
+    {
+        $this->createOrReplaceTable(
+            'table_test',
+            [
+                'id' => 'integer',
+                'date1' => 'datetime',
+                'date2' => 'date',
+            ],
+            [
+                [
+                    'id' => '1',
+                    'date1' => '1983-03-22 08:25:00',
+                    'date2' => '1983-03-22',
+                ],
+                [
+                    'id' => '2',
+                    'date1' => '1793-12-31 21:20:00',
+                    'date2' => '1793-12-31',
+                ],
+                [
+                    'id' => '3',
+                    'date1' => '2178-01-01 21:20:00',
+                    'date2' => '2178-01-01',
+                ],
+                [
+                    'id' => '4',
+                ],
+            ],
+        );
+    }
+
+    public function testAnonymizeWithRangeAsDate(): void
+    {
+        self::markTestSkipped();
+    }
+
+    public function testAnonymizeWithRangeAsDateHuge(): void
+    {
+        self::markTestSkipped();
+    }
```

Comment:
Why are those tests skipped ? And what is their purpose ?

Reply:
Done.